gems/smithy/lib/smithy/vise/endpoints.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative 'endpoints/built_in_binding'
+require_relative 'endpoints/function_binding'
+
+module Smithy
+  module Vise
```

Comment:
I don't think vise changes are necessary anymore. I think these binding classes probably are view classes.

Reply:
They don't really feel like view classes to me - at least in a traditional MVC sort of breakdown, I'd consider these more "model" related, which is why I had put them into vice.  

Additionally these are public extension points, so don't feel like they belong in views. 

Maybe they belong in the weld namespace?

Reply:
What would it look like just as an array of hash data? It doesn't look like we need objects at all. Can we just start simple?

Reply:
I think that makes this much messier as an extension point - having a well defined class makes it easy for users to extend.

Reply:
In practicality we will be the only ones adding endpoint bindings let's be real..

Reply:
I do generally prefer the classes, but I've removed them in the latest commit and removed the bindings from the plan.